.changeset/brown-singers-sit.md

```diff
@@ -0,0 +1,26 @@
+---
+'@openfn/language-dhis2': major
+---
+
+- Remove axios from `dhis2`.
+- Remove interceptor.
+- Allow users to pass in their own `headers` and `parseAs`.
+- Support `base64` format in the `get` function
+
+### Migration Guide
+
+- The `get` function has major changes in the parameters accepted.
```

Comment:
I will come back to this migration guide right at the end, but just to help my memory later, can you rephase to `the get function signature has changed. In particular, note that query parameters must be passed to request.options`

Don't worry about the language, I'll rework it later. It just helps me remember. Thanks!